authors/apps/share_article/views.py

```diff
@@ -0,0 +1,68 @@
+from django.urls import reverse
+from django_social_share.templatetags import social_share
+
+from rest_framework import status
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
```

Comment:
We agreed to use `generics.APIView`, because documentation appears to require this. (Get clarity from @mashafrancis). Refactor to reflect this.

Reply:
The documentation is reflecting the endpoint. No need to refactor